lte/gateway/release/build-magma.sh

```diff
@@ -123,7 +123,7 @@ MAGMA_DEPS=(
     "python3-pip"
     "python3-apt" # The version in pypi is abandoned and broken on stretch
     "python3-aioeventlet" # The version in pypi got deleted
-    "libsystemd-dev"
+    "libsystemd-dev >= 245.4-4ubuntu3.18"
```

Comment:
Out of interest, why do you fix the version everywhere else but have `>=` here?

Reply:
I wasn't entirely sure about the reasoning here either, but every other dependency that has a restricted version uses `>=`. I am not even sure if `fpm` wants a `==` or a single `=` for a fixed version.

Reply:
Also not, all the other version fixes are in `apt-get install` commands, which do not accept a range.

Reply:
Generally magma-preferences is not setup on systems where magma.deb is installed. Imho we do not need this here. Consequence is that magma.deb is not installable on magma-dev - as this needs some fidling anyway, I think this is ok for v1.8 -> should not be pinned